### DISCUSSION TITLE
More than a thousand portable Thonnys with **thonny-py5mode** installed out there?

### DISCUSSION THREAD

**Discussion by villares** [tabreturn/thonny-py5mode#65] [General]
Cheers @tabreturn and @hx2A,
 I'm not sure how meaningful this is, but there have been more that 1K downloads of the portable with py5 * plug-in in the last 3 months. These won't appear on @hx2A's PyPI statistics.

> ![image](https://github.com/user-attachments/assets/2efa0d37-d45d-480a-88c5-9aa9794ef3ed)
> https://tooomm.github.io/github-release-stats/?username=villares&repository=thonny-portable-with-py5

**Comment by hx2A** [comment]
Wow, that's great! There are even more installs than I thought. I do keep an eye on the  install numbers and am happy it is steadily growing.

Why doesn't the latest release have more installs than the one from 7/4/2024?

**Reply by villares** [reply]
I found that confusing too. Maybe I had some announcement before the first one? I find it unlikely... could it be PyCon US? Maybe the counter broke more recently...

**Comment by villares** [comment]
More than 2K portable Thonnys with thonny-py5mode and py5 now!

<img width="531" height="630" alt="image" src="https://github.com/user-attachments/assets/dc8db117-17e6-4d8d-974e-652a13d2fa07" />

**Reply by hx2A** [reply]
Excellent! That will grow even more once we finish the next plugin.
